rotkehlchen/assets/asset.py

```diff
@@ -459,6 +459,12 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xE41d2489571d322189246DaFA5ebDe1F4699F498'): 'ZRX',
 }
 
+WORLD_TO_NEXO = {
+    strethaddress_to_identifier('0xB62132e35a6c13ee1EE0f84dC5d40bad8d815206'): 'NEXONEXO',
```

Comment:
oh god ... :facepalm: 

NEXO -> NEXONEXO
GBP - > GBPX
USDT -> USDTERC

Reply:
and there will be more that we don't know yet :sweat: